app/main/views.py

```diff
@@ -111,7 +111,7 @@ def _gen_ind_standings(race_info, race_calendar):
 
     # A list of per-race points for each racer
     racer_race_points = {
-        racer_id: list((ri.points, ri.race_date) for ri in g)
+        racer_id: list((ri.points if not ri.points_dropped else '(%d)' % ri.points or 0, ri.race_date) for ri in g)
```

Comment:
Any thoughts on doing something similar for the Race view? Or maybe some other way in the Race view to denote that points have been dropped?

Reply:
Maybe, though I feel like it's not terribly interesting in that context -- it really only matters for overall season points, or if you're the person setting the dropped bit.